core/src/allocation.jl

```diff
@@ -386,35 +386,34 @@ function add_variables_flow!(
     return nothing
 end
 
-"""
-Add the basin allocation variables A_basin to the allocation problem.
-The variable indices are the allocation graph basin node IDs.
-Non-negativivity constraints are also immediately added to the basin allocation variables.
-"""
-function add_variables_allocation_basin!(
-    problem::JuMP.Model,
-    node_id_mapping::Dict{Int, Tuple{Int, Symbol}},
-    allocgraph_node_ids_basin::Vector{Int},
-)::Nothing
-    JuMP.@variable(problem, A_basin[i = allocgraph_node_ids_basin] >= 0.0)
-    return nothing
-end
-
-"""
-Add the user allocation constraints to the allocation problem:
-The flow to a user is bounded from above by the demand of the user.
-"""
-function add_constraints_user_allocation!(
+# """
+# Add the basin allocation variables A_basin to the allocation problem.
+# The variable indices are the allocation graph basin node IDs.
+# Non-negativivity constraints are also immediately added to the basin allocation variables.
+# """
+# function add_variables_allocation_basin!(
+#     problem::JuMP.Model,
+#     allocgraph_node_ids_basin::Vector{Int},
+# )::Nothing
+#     JuMP.@variable(problem, A_basin[i = allocgraph_node_ids_basin] >= 0.0)
+#     return nothing
+# end
+
```

Comment:
If this isn't used anymore I'd prefer to remove it, we can always take it out of git history if needed.